WORKSPACE.bazel

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A workspace for gapic-showcase
+"""
+workspace(name = "gapic_showcase")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+##
+# required by googleapis
+#
+http_archive(
+  name = "com_google_api_codegen",
+  strip_prefix = "gapic-generator-2.4.0",
+  urls = ["https://github.com/googleapis/gapic-generator/archive/v2.4.0.zip"],
+)
+
+##
+# googleapis for the common protos
+#
+http_archive(
+  name = "com_google_googleapis",
+  strip_prefix = "googleapis-cf41866c6f14f10a07aa1e2a1260fc0a2727d889",
+  urls = ["https://github.com/googleapis/googleapis/archive/cf41866c6f14f10a07aa1e2a1260fc0a2727d889.zip"],
+  sha256 = "943602b1e91621f282c1e790eb46d9a70e202fe65091b7d41415a43973cebec4"
+)
+load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
+switched_rules_by_language(name = "com_google_googleapis_imports", gapic = True)
+
+##
+# protobuf for the protobuf protos
```

Comment:
Can we simply update `gapic-showcase/schema/api-common-protos/WORKSPACE.bazel`?

Reply:
The showcase protos are not under the `api-common-protos` so we cannot use that `WORKSPACE.bazel` file for the showcase targets.

Reply:
Oh, right. 

(I suppose we could also put `.bazelrc` and `WORKSPACE` in `gapic-showcase/schema/`, but standard practice is to put it at the top of the repo so we can `BUILD` the repo. I'm fine with that. Thoughts, @noahdietz ?)